<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="layoutGWidget">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frameProperty">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="layoutGProperty">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="lineId"/>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelType">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QPushButton" name="buttonTips">
         <property name="text">
          <string>Tips</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QPushButton" name="buttonPrevious">
         <property name="text">
          <string>Previous</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLabel" name="labelId">
         <property name="text">
          <string>Id</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1" colspan="2">
        <widget class="QGroupBox" name="GBFilter">
         <property name="title">
          <string>Filter</string>
         </property>
         <layout class="QGridLayout" name="layoutGFilter">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QPlainTextEdit" name="PTEFilter"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="10" column="1" colspan="2">
        <widget class="QPushButton" name="buttonUpdate">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="CBType">
         <item>
          <property name="text">
           <string>Optional</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fullfill</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Judge</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Interlocution</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QPushButton" name="buttonAnswer">
         <property name="text">
          <string>Answer</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QPushButton" name="buttonNext">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLabel" name="labelTitle">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QPushButton" name="buttonAdd">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1" colspan="2">
        <widget class="QPushButton" name="buttonDelete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="lineTitle"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionQt"/>
    <addaction name="actionIndex"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/add.svg</normaloff>:/resources/icons/add.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/import.svg</normaloff>:/resources/icons/import.svg</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/export.svg</normaloff>:/resources/icons/export.svg</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/exit.svg</normaloff>:/resources/icons/exit.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionQt">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/qt.png</normaloff>:/resources/icons/qt.png</iconset>
   </property>
   <property name="text">
    <string>Qt</string>
   </property>
  </action>
  <action name="actionIndex">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/github.svg</normaloff>:/resources/icons/github.svg</iconset>
   </property>
   <property name="text">
    <string>Index</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/about.svg</normaloff>:/resources/icons/about.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
